Source/Upstream: Yes, in v238-stable branch

From b4467b2fb6a8e140700752a906ca4f98d5f75c74 Mon Sep 17 00:00:00 2001
From: "Matteo F. Vescovi" <mfvescovi@users.noreply.github.com>
Date: Tue, 20 Mar 2018 07:41:19 +0100
Subject: [PATCH 44/51] hwdb: ThinkPad 4x0s and X1 Carbon 3rd gen LEDs (#8492)

(cherry picked from commit c72102edd503cabfb7abeee009e06b823a9717c4)
---
 hwdb/60-keyboard.hwdb | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index f0976a6d4..808ed54a7 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -1422,6 +1422,14 @@ evdev:input:b0003v046Dp4002*
  KEYBOARD_LED_NUMLOCK=0
  KEYBOARD_LED_CAPSLOCK=0
 
+# Lenovo ThinkPad T430s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT430s
+ KEYBOARD_LED_CAPSLOCK=0
+
+# Lenovo ThinkPad T440s
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT440s
+ KEYBOARD_LED_CAPSLOCK=0
+
 # Lenovo ThinkPad T450s
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT450s
  KEYBOARD_LED_CAPSLOCK=0
@@ -1430,3 +1438,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT450s
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT560s
  KEYBOARD_LED_CAPSLOCK=0
  KEYBOARD_LED_NUMLOCK=0
+
+# Lenovo ThinkPad X1 Carbon 3rd Gen
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX1Carbon3rd
+ KEYBOARD_LED_CAPSLOCK=0
-- 
2.16.3

